nasa/harmony-regression-tests
# DAS-1530 - Update Trajectory Subsetter tests with required variables.

### Description:

This PR updates the Trajectory Subsetter regression tests to include the `shot_number` required variable in the GEDI L4A variable subset outputs. (All Trajectory Subsetter tests include this variable subset, as the granules are otherwise very large)

Note - when merged, these tests will fail until they are being run against v0.1.0 of the Trajectory Subsetter.

This PR also sneaks in a PR template for future pull requests.

### Jira Ticket:

DAS-1530

### Local test instructions:

* Build a version of the Trajectory Subsetter from the DAS-1530 branch using `./bin/build-image` in that repository (or `dev` if the changes have been merged already).
* Restart your local Harmony instance (using `./bin/restart-services` in the Harmony repository).
* Run the SDS Regression test notebook by either:
  * Pulling this branch of the repository.
  * Creating a conda environment with all the required packages.
  * Starting a local Jupyter notebook server.
  * Changing the `harmony_host_url` in the second code cell to "http://localhost:3000"
  * Running all cells - they should complete.
* Or:
  * Pulling this branch of the repository.
  * Using the `test/Makefile` to update the SDS regression test image (`make sds-image`).
  * Export an environment variable: `HARMONY_HOST_URL=http://localhost:3000`.
  * Using the `run_notebooks.sh` script. (For speed you could make it only run the SDS regression tests)

### Checklist:

* [ ] Ticket AC met
* [ ] Tests added/updates and passing
